myia/pipeline/standard.py

```diff
@@ -137,6 +137,15 @@
         '__eq__': operations.string_eq,
         '__ne__': operations.string_ne,
     },
+    xtype.Integral: {
+        '__and__': lop(operations.scalar_bit_and, xtype.Integral, '__and__'),
+        '__or__': lop(operations.scalar_bit_or, xtype.Integral, '__or__'),
+        '__xor__': lop(operations.scalar_bit_xor, xtype.Integral, '__xor__'),
+        '__lshift__': lop(
+            operations.scalar_bit_lshift, xtype.Integral, '__lshift__'),
+        '__rshift__': lop(
+            operations.scalar_bit_rshift, xtype.Integral, '__rshift__'),
+    },
```

Comment:
Now that we have `Integral`, I think we can also add the entries currently under `Int` and `UInt` in here, which will remove some code duplication.